lib/ansible/plugins/lookup/skydive.py

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
Having a lookup module is a much better idea than the existing `skydive_query` module.

Reply:
+1